recipe/emailverification/recipe.go

```diff
@@ -59,23 +58,23 @@ func RecipeInit(config schema.TypeInput) supertokens.RecipeListFunction {
 }
 
 func (r *Recipe) GetAPIsHandled() []supertokens.APIHandled {
-	generateEmailVerifyTokenAPI, _ := supertokens.NewNormalisedURLPath(emailverification.GenerateEmailVerifyTokenAPI)
-	emailVerifyAPI, _ := supertokens.NewNormalisedURLPath(emailverification.EmailVerifyAPI)
+	generateEmailVerifyTokenAPI, _ := supertokens.NewNormalisedURLPath(GenerateEmailVerifyTokenAPI)
+	emailVerifyAPI, _ := supertokens.NewNormalisedURLPath(EmailVerifyAPI)
 	return []supertokens.APIHandled{{
 		Method:                 "post",
 		PathWithoutAPIBasePath: *generateEmailVerifyTokenAPI,
-		ID:                     emailverification.GenerateEmailVerifyTokenAPI,
-		Disabled:               false, // doubt
+		ID:                     GenerateEmailVerifyTokenAPI,
+		Disabled:               r.APIImpl.GenerateEmailVerifyTokenPOST == nil,
```

Comment:
`GenerateEmailVerifyTokenPOST` is not a pointer in APIImterface.. so `GenerateEmailVerifyTokenPOST == nil` will always be false?